test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NetworkCallCountTest.cs

```diff
@@ -24,13 +24,20 @@ public class NetworkCallCountTest
         public void NetworkCallCount_RestoreLargePackagesConfigWithMultipleSourcesWithAllMissingPackages()
         {
             // Arrange
-            Util.ClearWebCache();
             var testCount = 100;
 
             using (var server2 = new MockServer())
             using (var server1 = new MockServer())
             using (var workingPath = TestDirectory.Create())
             {
+                // Override default http-cache using the NUGET_HTTP_CACHE_PATH environment variable.
```

Comment:
Consider using https://github.com/NuGet/NuGet.Client/blob/dev/test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestPathContext.cs instead of TestDirectory.
It creates a folder with a NuGet.Config and isolates it correctly. 

It also has a prebuild structure for solution, projects, the global packages folder etc.